src/daemon/environment.c

```diff
@@ -94,6 +127,11 @@ void set_environment_for_plugins_and_scripts(void) {
     // disable buffering for python plugins
     setenv("PYTHONUNBUFFERED", "1", 1);
 
+    p = getenv("CURL_CA_BUNDLE");
+    if(!p || !*p) p = detect_libcurl_default_ca();
```

Comment:
We want libcurl’s default to be our fallback, not our preferred option. It may or may not match up with what the TLS implementation does, but for consistency reasons we need it to match up with the TLS implementation (if we don’t, then we have the possibility of being able to claim an agent but not establish an ACLK connection fully, because the ACLK code _doesn’t_ use libcurl and instead just goes with what the TLS implementation gives it).